cli/cmd/generate.go

```diff
@@ -63,6 +63,7 @@ subcommands.`,
 	cmd.Flags().StringP("policy", "p", rulesFilename, "path to policy (.rego) file")
 	cmd.Flags().StringP("settings", "s", settingsFilename, "path to settings (.json) file")
 	cmd.Flags().StringP("manifest", "m", manifestFilename, "path to manifest (.json) file")
+	cmd.Flags().String("reference-values", "default", "set the default reference values used for attestation (one of: default|aks)")
```

Comment:
```suggestion
	cmd.Flags().String("reference-values", "", "set the default reference values used for attestation (one of: aks)")
```
If default isn't a case we want to explicitly select, I'd opt to use default when flag isn't set instead of making it a string opt.